from PySide6 import QtCore, QtGui, QtWidgets


class Ui_NewItem(object):
    def setupUi(self, NewItem):
        NewItem.setObjectName("NewItem")
        NewItem.resize(343, 370)
        self.lblCreateTitle = QtWidgets.QLabel(NewItem)
        self.lblCreateTitle.setGeometry(QtCore.QRect(70, 20, 221, 41))
        font = QtGui.QFont()
        font.setPointSize(20)
        self.lblCreateTitle.setFont(font)
        self.lblCreateTitle.setObjectName("lblCreateTitle")
        self.widget = QtWidgets.QWidget(NewItem)
        self.widget.setGeometry(QtCore.QRect(10, 70, 321, 281))
        self.widget.setObjectName("widget")
        self.formLayout = QtWidgets.QFormLayout(self.widget)
        self.formLayout.setContentsMargins(0, 0, 0, 0)
        self.formLayout.setObjectName("formLayout")
        self.lblItemName = QtWidgets.QLabel(self.widget)
        self.lblItemName.setObjectName("lblItemName")
        self.formLayout.setWidget(0, QtWidgets.QFormLayout.LabelRole, self.lblItemName)
        self.lineEditItemName = QtWidgets.QLineEdit(self.widget)
        font = QtGui.QFont()
        font.setPointSize(11)
        self.lineEditItemName.setFont(font)
        self.lineEditItemName.setObjectName("lineEditItemName")
        self.formLayout.setWidget(0, QtWidgets.QFormLayout.FieldRole, self.lineEditItemName)
        self.lblUser = QtWidgets.QLabel(self.widget)
        self.lblUser.setObjectName("lblUser")
        self.formLayout.setWidget(1, QtWidgets.QFormLayout.LabelRole, self.lblUser)
        self.lineEditUser = QtWidgets.QLineEdit(self.widget)
        self.lineEditUser.setObjectName("lineEditUser")
        self.formLayout.setWidget(1, QtWidgets.QFormLayout.FieldRole, self.lineEditUser)
        self.lblPassword = QtWidgets.QLabel(self.widget)
        self.lblPassword.setObjectName("lblPassword")
        self.formLayout.setWidget(2, QtWidgets.QFormLayout.LabelRole, self.lblPassword)
        self.lineEditPassword = QtWidgets.QLineEdit(self.widget)
        self.lineEditPassword.setObjectName("lineEditPassword")
        self.formLayout.setWidget(2, QtWidgets.QFormLayout.FieldRole, self.lineEditPassword)
        self.lblUrl = QtWidgets.QLabel(self.widget)
        self.lblUrl.setObjectName("lblUrl")
        self.formLayout.setWidget(3, QtWidgets.QFormLayout.LabelRole, self.lblUrl)
        self.lineEditUrl = QtWidgets.QLineEdit(self.widget)
        self.lineEditUrl.setObjectName("lineEditUrl")
        self.formLayout.setWidget(3, QtWidgets.QFormLayout.FieldRole, self.lineEditUrl)
        self.lblCategory = QtWidgets.QLabel(self.widget)
        self.lblCategory.setObjectName("lblCategory")
        self.formLayout.setWidget(4, QtWidgets.QFormLayout.LabelRole, self.lblCategory)
        self.cmbBoxCategory = QtWidgets.QComboBox(self.widget)
        self.cmbBoxCategory.setObjectName("cmbBoxCategory")
        self.formLayout.setWidget(4, QtWidgets.QFormLayout.FieldRole, self.cmbBoxCategory)
        self.lblItemDescription = QtWidgets.QLabel(self.widget)
        self.lblItemDescription.setObjectName("lblItemDescription")
        self.formLayout.setWidget(5, QtWidgets.QFormLayout.LabelRole, self.lblItemDescription)
        self.textEditItemDescription = QtWidgets.QTextEdit(self.widget)
        self.textEditItemDescription.setSizeAdjustPolicy(QtWidgets.QAbstractScrollArea.AdjustToContentsOnFirstShow)
        self.textEditItemDescription.setObjectName("textEditItemDescription")
        self.formLayout.setWidget(5, QtWidgets.QFormLayout.FieldRole, self.textEditItemDescription)
        self.btnSaveClose = QtWidgets.QPushButton(self.widget)
        self.btnSaveClose.setObjectName("btnSaveClose")
        self.formLayout.setWidget(6, QtWidgets.QFormLayout.FieldRole, self.btnSaveClose)

        self.retranslateUi(NewItem)
        QtCore.QMetaObject.connectSlotsByName(NewItem)

    def retranslateUi(self, NewItem):
        _translate = QtCore.QCoreApplication.translate
        NewItem.setWindowTitle(_translate("NewItem", "Dialog"))
        self.lblCreateTitle.setText(_translate("NewItem", "Create a new item"))
        self.lblItemName.setText(_translate("NewItem", "Item name: "))
        self.lineEditItemName.setPlaceholderText(_translate("NewItem", "Type name company or service"))
        self.lblUser.setText(_translate("NewItem", "User:"))
        self.lblPassword.setText(_translate("NewItem", "Password:"))
        self.lblUrl.setText(_translate("NewItem", "URL:"))
        self.lblCategory.setText(_translate("NewItem", "Category:"))
        self.lblItemDescription.setText(_translate("NewItem", "Description:"))
        self.btnSaveClose.setText(_translate("NewItem", "Save and close"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    NewItem = QtWidgets.QDialog()
    ui = Ui_NewItem()
    ui.setupUi(NewItem)
    NewItem.show()
    sys.exit(app.exec_())
